design-proposals/isolate-emulator-thread-with-smt-fix/isolate-emulator-thread-with-smt-fix.md

```diff
@@ -155,13 +198,20 @@ Will add unit tests to `renderresources_test.go` to check extra housekeeping log
 # Implementation Phases
 ## Phase 1: Align CPU to SMT requirements when feature gate is enabled and annotation is set on the VMI.
 * Virt-config:
-  * Add support for the new feature gate: `FullPCPUsOnly`.
+  * Add support for the new feature gate: `EmulatorThreadEvenParity`.
 * Virt-API:
-  * Add support for the new alpha annotation: `kubevirt.io/CPUManagerPolicyBetaOptions:full-pcpus-only`.
+  * Add support for the new alpha annotation: `kubevirt.io/EmulatorThreadEvenParity:`.
+  * in case annotation is set on kubevirt CR, the feature gate is enabled and the isolateEmulatorThread is enabled:
+    * the VMI mutating-webhook will copy the kubevirt CR annotation to the VMI annotations.
 * Virt-controller:
-  * In case annotation is set, feature gate is enabled and IsolateEmulatorThread is true, add logic to Align the total number of CPUs to be even.
+  * in case annotation is set on kubevirt CR, the feature gate is enabled and the isolateEmulatorThread is enabled:
```

Comment:
Please see the comment above about performing this set in the VMI creation admitter.

Reply:
talked offline, DONE.